src/test/java/org/example/adapter/out/InMemoryInvoiceRepoTest.java

```diff
@@ -0,0 +1,174 @@
+package org.example.adapter.out;
+
+import org.example.DataCreator;
+import org.example.domain.customer.Customer;
+import org.example.domain.invoice.CostInvoice;
+import org.example.domain.invoice.IncomeInvoice;
+import org.example.domain.invoice.Invoice;
+import org.example.domain.money.Money;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.time.YearMonth;
+import java.util.HashSet;
+
+class InMemoryInvoiceRepoTest {
+
+    private final InMemoryInvoiceRepo inMemoryInvoiceRepo = new InMemoryInvoiceRepo();
+
+
+    @Test
+    void shouldCreateInvoicesCorrectly() {
+        //given
+        Invoice invoice1 = DataCreator.createCostInvoice1();
```

Comment:
testowanie repo jest zbedne testujesz tutja jave

Reply:
I am a beginner - wanted to check if my way of implementing stuff is correct.
Resolved